pkg/apis/kubeone/validation/validation.go

```diff
@@ -60,6 +60,7 @@ func ValidateCloudProviderSpec(p kubeone.CloudProviderSpec, fldPath *field.Path)
 
 	switch p.Name {
 	case kubeone.CloudProviderNameAWS:
+	case kubeone.CloudProviderNameAzure:
```

Comment:
Is `cloudConfig` required for Azure? If yes, should we enforce it here?

Reply:
yeah... actually it's required for vSphere too and I don't remember that I've done it 😅

Reply:
done